### ISSUE TITLE
install  fails

### ISSUE THREAD

**Issue by JulianXin** [influxdata/influxdb-relay#79]
go get -u github.com/influxdata/influxdb-relay
package github.com/influxdata/influxdb/v2/pkg/escape: cannot find package "githu                                                                             b.com/influxdata/influxdb/v2/pkg/escape" in any of:
        /usr/local/go/src/github.com/influxdata/influxdb/v2/pkg/escape (from $GO                                                                             ROOT)
        /root/go/src/github.com/influxdata/influxdb/v2/pkg/escape (from $GOPATH)

Centos7.5    golang:1.5.4(I also have tried 1.7.6 and  1.14.2)

**Comment by russorat** [comment]
@JulianXin thanks for the issue. not a lot of activity in this repo lately, but maybe someone from the community can help here.

**Comment by vgrozev** [comment]
It seems that refactoring would be the problem for influx v2, but the current one for Centos is v1.8. 
Does anyone know what can be done to actually install influxdb-relay on Centos at the moment?

**Comment by jsternberg** [comment]
There's two options for this. We either update the readme or just add a `go.mod` file.

I kind of favor adding `go.mod` but I'm hesitant to change any code or dependencies.

If you want to install this with a more recent version of Go, you'll have to add `GO111MODULE=off` as an environment variable when building. You will also need to use godeps to retrieve the specific versions of the dependencies.

**Comment by karalov** [comment]
I have the same issue, GO111MODULE=off didn't help. What should I do with godeps, can you please send exact steps how can I install relay? Any workaround will help. Thank you.
# go get -u github.com/influxdata/influxdb-relay
/opt/go/src/github.com/influxdata/influxdb/models/points.go:18:2: code in directory /opt/go/src/github.com/influxdata/influxdb/pkg/escape expects import "github.com/influxdata/influxdb/pkg/escape"

**Comment by vgrozev** [comment]
yes, GO111MODULE=off, does not to fix it for me either, so I cannot get the code in order to use 'godeps' on it. 
... on the other hand I am not in depth familiar with GO, so I might be doing something wrong.

**Comment by rockyluke** [comment]
You should give a try to https://github.com/veepee-oss/influxdb-relay.

**Comment by jsternberg** [comment]
@vgrozev I just merged a change to add a `go.mod` file to this repository so that the install would be easier. Can you give it another try? This time the `GO111MODULE` value should be either not set or set to `on`.

@rockyluke thanks for bringing that fork to our attention! We're giving it an active look to see how we could better integrate community changes to the upstream repository.

**Comment by vgrozev** [comment]
@jsternberg Thanks, it seems that it installs ok now and the systemd process I have seem to work OK. 
Thanks. 
@rockyluke I looked at that fork in the past, but although it seems they are adding cool features and it is actively maintained, the config file syntax is changed so much that it will require a lot of work on my side to accommodate at the moment.

**Comment by rockyluke** [comment]
@jsternberg I will be happy to help you backporting some simple features we added.

@vgrozev sadly we had to change the whole configuration file 😢 on the other hand we implemented some nice features.

**Comment by xiuqinzhao** [comment]
tried all suggestions, none of them is working. Please help out here

**Comment by karalov** [comment]
> tried all suggestions, none of them is working. Please help out here

Relay is not supported anymore and is not recommended by influx support. Try to use telegraf instead.

**Comment by jhg03a** [comment]
> Relay is not supported anymore and is not recommended by influx support.

Is there an official statement on that?  Telegraf and the influxdb-relay do very different things.
